src/control/lib/control/response.go

```diff
@@ -36,10 +36,6 @@ type (
 	// HostResponseChan defines a channel of *HostResponse items returned
 	// from asynchronous unary RPC invokers.
 	HostResponseChan chan *HostResponse
-
-	// HostResponseReport defines the function signature for a callback
-	// invoked when a host response is received.
-	HostResponseReportFn func(*HostResponse)
```

Comment:
Bad merge?

Reply:
Yes.  I think this file got a update without the copyright date being changed and that messed up the merge.

Will fix.